include/zephyr/bluetooth/audio/pacs.h

```diff
@@ -45,6 +45,43 @@ struct bt_pacs_cap {
 	sys_snode_t _node;
 };
 
+/** Structure for registering PACS */
+struct bt_bap_pacs_register_param {
+#if defined(CONFIG_BT_PAC_SNK)
```

Comment:
I think you need to add

```suggestion
#if defined(CONFIG_BT_PAC_SNK) || defined(__DOXYGEN__)
```

For the fields to appear in the documentation, but let's see when the doc build finishes if they are probably shown there or not

Reply:
Yeah, looks like we need it: https://builds.zephyrproject.io/zephyr/pr/83730/docs/doxygen/html/structbt__bap__pacs__register__param.html#details

Otherwise the struct is empty in the documentation